doc/api/cli.md

```diff
@@ -1577,6 +1577,43 @@ on the number of online processors.
 If the value provided is larger than V8's maximum, then the largest value
 will be chosen.
 
+### `--watch`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Starts Node.js in watch mode.
+When in watch mode, changes in the watched files cause the Node.js process to
+restart.
+By default, watch mode will watch the entry point
+and any required or imported module.
+Use `--watch-path` to specify what paths to watch.
+
+This flag cannot be combined with
+`--check`, `--eval`, `--interactive`, or the REPL.
+
+### `--watch-path`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Specify what paths to watch in watch mode.
+This will turn off watching of required or imported modules.
+
+```console
+$ node --watch-path=./src index.js
+```
```

Comment:
What happens if `--watch-path` is supplied but `--watch` is not? I would expect that nothing happens, correct? In any case the docs should specify it.

Reply:
I added an `Implies("--watch-path", "--watch");` so it is enough to add `--watch-path` only - I will document that fact